tests/modules/transformer/bimodal_attention_test.py

```diff
@@ -49,7 +49,7 @@ def test_forward_runs(self):
 
         self.biattention.forward(
             torch.randn(2, 3, 6),
-            torch.randn(2, 2, 3, 3),
             torch.randn(2, 3, 4),
             torch.randn(2, 2, 3, 3),
+            torch.randn(2, 2, 3, 3),
         )
```

Comment:
Why did this change?

Reply:
I changed the order of the arguments in bimodal_attention, to allow for masks to have default `None` values.